posthog/tasks/status_report.py

```diff
@@ -80,10 +94,12 @@ def status_report(*, dry_run: bool = False) -> Dict[str, Any]:
 def capture_event(name: str, report: Dict[str, Any], dry_run: bool) -> None:
     if not dry_run:
         posthoganalytics.api_key = "sTMFPsFhdP1Ssg"
-        disabled = posthoganalytics.disabled
-        posthoganalytics.disabled = False
-        posthoganalytics.capture(get_machine_id(), "instance status report", report)
-        posthoganalytics.disabled = disabled
+        posthoganalytics.capture(get_machine_id(), name, {**report, "scope": "machine"})
+
+        for user in User.objects.all():
+            posthoganalytics.capture(user.distinct_id, name, {**report, "scope": "user"})
+    else:
+        print(name, json.dumps(report))  # noqa: T001
```

Comment:
@paolodamico Code looks good but question about the event ––

Wonder if we should send a separate event for the users? 

e.g. 'instance user report' or something and it can be just an event for that user, as a prop we pass in an instance ID or team ID (whatever makes most sense).

I think it would just be nice to look at Uniques across instance status report and know that's how many instances we have and not need to conflate it with users